_studio/mfx_lib/shared/include/mfx_brc_common.h

```diff
@@ -160,6 +160,21 @@ class AVGBitrate
 #define MAX_RACA 361.0
 #define RACA_SCALE 128.0
 
+/*
+NalHrdConformance | VuiNalHrdParameters   |  Result
+--------------------------------------------------------------
+    off                  any                => MFX_BRC_NO_HRD
+    default              off                => MFX_BRC_NO_HRD
+    on                   off                => MFX_BRC_HRD_WEAK
+    on (or default)      on (or default)    => MFX_BRC_HRD_STRONG
+--------------------------------------------------------------
+*/
+enum : mfxU16
+{
+    MFX_BRC_NO_HRD,
```

Comment:
You are highly relying on default values start from zero, but not specifying it explicitly.  Better to add initialization

```MFX_BRC_NO_HRD = 0,```

Reply:
changed